File: Go LeetCode/Find the Difference of Two Arrays.go
package main

func findDifference(nums1 []int, nums2 []int) [][]int {
	nums1Map := make(map[int]bool)
	nums2Map := make(map[int]bool)

	for _, num := range nums1 {
		nums1Map[num] = true
	}
	for _, num := range nums2 {
		nums2Map[num] = true
	}

	nums1Only := []int{}
	nums2Only := []int{}

	for num := range nums1Map {
		if !nums2Map[num] {
			nums1Only = append(nums1Only, num)
		}
	}

	for num := range nums2Map {
		if !nums1Map[num] {
			nums2Only = append(nums2Only, num)
		}
	}

	return [][]int{nums1Only, nums2Only}
}

File: Python LeetCode/Find the Difference of Two Arrays.py
from typing import List


class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        check :set = set(nums1).intersection(set(nums2))
        return [[i for i in nums1 if i not in check], [i for i in nums2 if i not in check]]